ports/stm/boards/STM32F746_fs.ld

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Please rename this file as the STM32F746xG to denote it's for the 1MB version. It's irritating that the HAL doesn't provide good recognition of this difference.